.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Bug report
+about: Create a report to help us improve
+labels: bug
+
+---
+
+<!-- 🚨 STOP 🚨 STOP 🚨 STOP 🚨
```

Comment:
When comparing this "bug report" and "question", this one is part of the comment block and the other is not.  Do we want to make it consistent?

Reply:
In "bug report", the "stop" line is just in the comment - we don't want it to be visible when the issue is created.

I thought it may be better to render it visible in "question" and "security vulnerability" issues.

On the second thought, I guess it does not make sense. If the issue is created despite the fact that we asked people not to, then there is little value in having a big "stop" alert at the top.

I'll change the templates to always put the "stop" alert into a comment.